bundles/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/internal/handler/SeneyeHandler.java

```diff
@@ -69,21 +69,50 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void newState(SeneyeDeviceReading readings) {
         if (readings != null) {
-            updateState(CHANNEL_TEMPERATURE, new DecimalType(readings.temperature.curr));
-            updateState(CHANNEL_NH3, new DecimalType(readings.nh3.curr));
-            updateState(CHANNEL_NH4, new DecimalType(readings.nh4.curr));
-            updateState(CHANNEL_O2, new DecimalType(readings.o2.curr));
-            updateState(CHANNEL_PAR, new DecimalType(readings.par.curr));
-            updateState(CHANNEL_PH, new DecimalType(readings.ph.curr));
-            updateState(CHANNEL_LUX, new DecimalType(readings.lux.curr));
-            updateState(CHANNEL_KELVIN, new DecimalType(readings.kelvin.curr));
-            updateState(CHANNEL_LASTREADING, new DateTimeType(readings.status.getLast_experimentDate()));
-            updateState(CHANNEL_SLIDEEXPIRES, new DateTimeType(readings.status.getSlide_expiresDate()));
-            updateState(CHANNEL_WRONGSLIDE, new StringType(readings.status.getWrong_slideString()));
-            updateState(CHANNEL_SLIDESERIAL, new StringType(readings.status.getSlide_serialString()));
-            updateState(CHANNEL_OUTOFWATER, new StringType(readings.status.getOut_of_waterString()));
-            updateState(CHANNEL_DISCONNECTED, new StringType(readings.status.getDisconnectedString()));
+            if (seneyeService.seneyeType == 3) {
+                logger.debug("found that sensor is type 3");
```

Comment:
Maybe log what type 1,2 or 3 actually is ? Pond/Home/Reef

Reply:
Hi Niko,

Many thanks for the feedback. I have changed the log as suggested.
This binding has been great for me, thanks for creating it.